web/static/css/components/_Achievement.scss

```diff
@@ -0,0 +1,20 @@
+.Achievement {
+  margin-bottom: $vertical-spacing;
```

Comment:
Properties should be ordered display, margin-bottom, text-align

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)